include/nbl/video/utilities/CScanner.h

```diff
@@ -8,15 +8,14 @@
 
 #include "nbl/video/IPhysicalDevice.h"
 #include "nbl/video/utilities/IDescriptorSetCache.h"
+#include "nbl/video/utilities/CArithmeticOps.h"
 
+#include "nbl/builtin/hlsl/scan/declarations.hlsl"
+static_assert(NBL_BUILTIN_MAX_LEVELS & 0x1, "NBL_BUILTIN_MAX_LEVELS must be odd!");
```

Comment:
if we don't have our own `static_assert` in HLSL, then stick this inside a `ifndef __HLSL_VERSION` inside the HLSL header